### PR TITLE
changing telegram bot to use webhook instead of updates api

### PR DESCRIPTION
**Pull request by tkalir**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -16,6 +16,10 @@ services:
       - TWITTER_ACCESS_SECRET
       - GOOGLE_APPLICATION_CREDENTIALS=GOOGLE_APPLICATION_CREDENTIALS_KEY.json
       - MAPS_API_KEY
+      - AWS_ACCESS_KEY=${DBRESTORE_AWS_ACCESS_KEY_ID}
```
Comment (atalyaalon) [inline docker-compose.yml:19]: @tkalir is this change necessary? If it it simply for local debugging?

File: docker-compose.yml [hunk 2]
```diff
@@ -16,6 +16,10 @@ services:
...
+      - AWS_SECRET_KEY=${DBRESTORE_AWS_SECRET_ACCESS_KEY}
```
Comment (atalyaalon) [inline docker-compose.yml:20]: Same here?

**Comment by OriHoch** [comment]
as long as the secret values are not committed to git, and kept in .env file or something like that, it's fine

**Comment by atalyaalon** [comment]
@tkalir see Ori's answer, you can add the vars (w/o the secrets of course) to our docker-compose files
